kmymoneysettings.h is generated by the kmm_settings target. We need to ensure
that it is built before kmm_ofximport_dialogs.

kmymoney/plugins/ofximport/dialogs/kofxdirectconnectdlg.cpp:19:10: fatal error: 'kmymoneysettings.h' file not found

--- kmymoney/plugins/ofximport/dialogs/CMakeLists.txt.orig	2018-06-08 00:37:49 UTC
+++ kmymoney/plugins/ofximport/dialogs/CMakeLists.txt
@@ -15,3 +15,5 @@ set(kmm_ofximport_dialogs_UI
 kde4_add_ui_files(kmm_ofximport_dialogs_la_SOURCES
   ${kmm_ofximport_dialogs_UI})
 kde4_add_library(kmm_ofximport_dialogs STATIC ${kmm_ofximport_dialogs_la_SOURCES})
+
+add_dependencies(kmm_ofximport_dialogs kmm_settings)
